src/plugins/opensearch_dashboards_react/public/markdown/markdown.tsx

```diff
@@ -37,23 +37,23 @@ import { getSecureRelForTarget } from '@elastic/eui';
 import './index.scss';
 /**
  * Return a memoized markdown rendering function that use the specified
- * whiteListedRules and openLinksInNewTab configurations.
- * @param {Array of Strings} whiteListedRules - white list of markdown rules
+ * allowListedRules and openLinksInNewTab configurations.
+ * @param {Array of Strings} allowListedRules - allow list of markdown rules
  * list of rules can be found at https://github.com/markdown-it/markdown-it/issues/361
  * @param {Boolean} openLinksInNewTab
  * @return {Function} Returns an Object to use with dangerouslySetInnerHTML
  * with the rendered markdown HTML
  */
 export const markdownFactory = memoize(
-  (whiteListedRules: string[] = [], openLinksInNewTab: boolean = false) => {
+  (allowListedRules: string[] = [], openLinksInNewTab: boolean = false) => {
```

Comment:
I don't think this is heavily used in the project but should we be supporting whiteListedRules annotate with `@deprecated` and allow for allowListedRules